.\install.ps1 failed

I followed the [setup instructions](https://github.com/awslabs/aws-saas-boost/blob/main/docs/getting-started.md#step-1---set-up-tooling)  with one exception -- my machine already had `node v14.16.1` on it, so I did not install `node v14.15`.
```
.\install.ps1
AWS Region = us-east-2
Building Java Installer with maven
Java Installer build completed
Downloading dependencies for React Web App, please be patient
Download dependencies completed for React Web App
Launch Java Installer for SaaS Boost
Error: Could not find or load main class .util.logging.config.file=logging.properties
Caused by: java.lang.ClassNotFoundException: /util/logging/config/file=logging/properties
```




### Environment

  - **AWS Region  :** us-east-2
  - **AWS SaaS Boost Version :** commit 74e9047ce3f06779c645db392236c6dd7fe6fac9 (HEAD -> main, origin/main, origin/HEAD)
  - **Workload OS  (Linux or Windows) :** Windows

### Other

<!-- e.g. detailed explanation, stacktraces, related issues, suggestions on how to fix, links for us to have context, eg. associated pull-request, stackoverflow, slack, etc -->




--- 

This is :bug: Bug Report

--- a/install.ps1
+++ b/install.ps1
@@ -135,4 +135,4 @@ Write-host "Launch Java Installer for SaaS Boost"
 
 $env:AWS_REGION = $AWS_REGION
 
-java -Djava.util.logging.config.file=logging.properties -jar ${CURRENT_DIR}\installer\target\SaaSBoostInstall-1.0.0-shaded.jar
+java "-Djava.util.logging.config.file=logging.properties" -jar ${CURRENT_DIR}\installer\target\SaaSBoostInstall-1.0.0-shaded.jar

